### ISSUE TITLE
The code provided in the code along doesn't really work.

### ISSUE THREAD

**Issue by nothingisfunny** [learn-co-students/integrating-react-and-redux-codealong-v-000#40]
This only worked after removing `;` on line 10 of changeCounter and after passing property to Counter component `<Counter store={this.props.store}/>`. Not sure if the purpose of this code along was to **not** pass the properties like that?

**Comment by jminterwebs** [comment]
This is still an issue

**Comment by howardbdev** [comment]
agreed.  The problem is that if you just follow the lesson, within `Counter.js`, `props.store.getState()` returns undefined until the very-very end of the codealong. I was able to work around it using
 `<div>{props.store.getState() ? props.store.getState().count : 0}</div>` 
until the end of the lesson, at which point it works and the above can be changed back to just
`<div>{props.store.getState().count}</div>`

**Comment by samsonyuwono** [comment]
This is still an issue. Can we get this lesson fixed?

**Comment by lisaychuang** [comment]
Still an issue! @howardbdev solution above works, this lesson should definitely be updated.

I've opened a Pull Request to update code [#135](https://github.com/learn-co-students/integrating-react-and-redux-codealong-v-000/pull/135)

**Comment by IngridWong0715** [comment]
Please fix the issue! We're in May 2018 and the codealong still doesn't work until the very end which is incredibly confusing.
